### DISCUSSION TITLE
MDX - Assistance

### DISCUSSION THREAD

**Discussion by GoDonks** [mdx-js/mdx#2470] [Q&A] [answered]
Hi.  I'm stuck as to why this code is not working:
`CASE
     WHEN
     (STRTONUM(SUBSTRING([FY24], 3, 2)) >= 0)
     THEN 999
     ELSE Missing
END  `

I need it to resolve to 24 and be a number.  Error in Essbase is returning:  Syntax error in input MDX query on line 3 at token ',' .  Any help is much appreciated.  Maybe I'm overlooking something obvious.

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @GoDonks! 👋 
Sorry you ran into confusion.

This is the forum for MDX the Markdown language.
Not MDX the Oracle SQL functions https://docs.oracle.com/en/database/other-databases/essbase/21/esscq/mdx-function-list.html
Oracle cloud has their own support at https://www.oracle.com/support/
